### DISCUSSION TITLE
to access the pickle file which is stored in git lfs

### DISCUSSION THREAD

**Discussion by Harshavardhanmashetti** [git-lfs/git-lfs#5681] [Q&A]
I am cloning a project. when I was working on that project the issue come across me is in the pickle file it contains
`version https://git-lfs.github.com/spec/v1
oid sha256:48c0600b2878650d731d22e91bccf234019813255d6ae93b930851a0e73f0089
size 574495608`
my question how to access or download the pickle file in the git-lfs

**Comment by bk2204** [comment]
By default, if you've already run `git lfs install`, Git should turn pointer files (like the contents you have) into the large objects they represent.  If you haven't, then you need to run `git lfs install --skip-repo`, which will update `~/.gitconfig`, and then you can run `git lfs pull` to check out the large files.

As a note, untrusted Pickle files can execute arbitrary code just by being loaded, as mentioned [in the Python documentation](https://docs.python.org/3/library/pickle.html).  I mention this because it sounds like you're not the author of the repository or pickle file in question.
